docs/tools/CLI/cli-device-management.md

```diff
@@ -55,7 +56,7 @@ $ mbed compile
 ```
 
 This generates a payload to update the device with. After generating the payload, update the device through Device Management with:
-
+<!---We talk about an image rather than a payload.
```

Comment:
Because this is the only place we talk about "payload" (at least in Mbed OS), I think we should standardize to match Device Management. @iriark01 , what do you think?

Reply:
Is payload clearer? Although I guess that ship's sailed, on the DM side

Reply:
This might be one of those instances where both are in the code. I'll poke around.

Reply:
Yes--payload is in the SDKs and various references throughout; it seems to refer more to the size of a block of information that you transmit, rather than the firmware *image* as such. Though there looks to be enough overlap that we still have competing terms for the same thing.

Reply:
This sounds like something we should confirm with engineering.

Reply:
@yogpan01, do you have an opinion on this?